g/gopsutil/gopsutil_ubi_8.5.sh

```diff
@@ -0,0 +1,49 @@
+# -----------------------------------------------------------------------------
+#
+# Package	: github.com/shirou/gopsutil/v3
+# Version	: v3.21.1
+# Source repo	: https://github.com/shirou/gopsutil
+# Tested on	: RHEL 8.3
+# Language      : GO
+# Travis-Check  : False
```

Comment:
@atharv-phadnis based on the value of this flag and "Tested On" field, it seems that the script was tested in a VM env. and probably not on a UBI base? if so, can you please rename the script accordingly and mention in comments that it needs a VM env. to test on and why

Reply:
Renamed the file, added an "Environment" field in comments to specify testing environment